pyopia/instrument/silcam.py

```diff
@@ -51,6 +51,90 @@ def load_mono8(filename):
     return img
 
 
+def load_bayer_rgb8(filename):
+    '''load an RG8 .bsilc file from disc
+    
+    Assumes 8-bit Bayer-RG (Red-Green) image in range 0-255
+
+    Parameters
+    ----------
+    filename : string
+        filename to load
+
+    Returns
+    -------
+    array
+        raw image float between 0-1
+    '''
+    img_bayer = np.load(filename, allow_pickle=False)[:,:,0].astype(np.int16)
```

Comment:
Lets check the implication of using int16 here, since we are expecting the raw data to be 8-bit, and want an output of floats in range 0-1